docs/variants/pc_engines/building-manual.md

```diff
@@ -0,0 +1,56 @@
+# Building manual
+
+## Intro
+
+This document describes the procedure for compiling Dasharo for PC Engines
+apu2 series platform.
+
+## Requirements
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
+## Build Dasharo firmware
+
+=== "UEFI firmware"
```

Comment:
Are we planning to call it UEFI, SeaBIOS, or Heads firmware for consistency of naming? Or maybe we should use names from DMI Dasharo (coreboot+UEFI), Dasharo (coreboot+Heads) etc.?

Reply:
I think option 2 is better

Reply:
https://github.com/Dasharo/docs/pull/794/commits/d5dcf62a7a05bbc9cd6efeb189d1cc9c6953d2a2
https://github.com/Dasharo/docs/pull/794/commits/d737a9610ee8cc3237969a7fe7bba2948700009b